docs/src/main/sphinx/release/release-446.md

```diff
@@ -0,0 +1,67 @@
+# Release 446 (29 Apr 2024)
+
+## General
+
+* Improve performance of `INSERT` statements into partitioned tables when the
+  `retry_policy` configuration property is set to `TASK`. ({issue}`21661 `)
+* Reduce delay before stopping queries when the cluster runs out of memory. ({issue}`21719`)
+* Fix catalog names including `"` marks when creating them with `CREATE CATALOG`
+  statements. ({issue}`21399`)
+* Fix query failure when a token in a query ends with a `:`. ({issue}`21676`)
+* Fix potential query failure for queries using [SQL routines](/routines) with
+  the `LEAVE`, `ITERATE`, `REPEAT`, or `WHILE` labels. ({issue}`21682`)
```

Comment:
those are not labels .. you can place labels with them though.. so need reword probably